apps/02RockPaperScissors.js

```diff
@@ -7,10 +7,41 @@ prompt.start();
 
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
+var hand1 = hand1.toLowerCase();
+var hand2 = hand2.toLowerCase();
 
+
+
+if (hand1 === hand2) {
+return "It's a tie!";
 }
 
+if (hand1 === 'rock') {
+  if (hand2 === 'scissors') {
+   return 'Hand one wins!';
+  } else {
+  return 'Hand two wins!';
+ }
+ }
+// If we reach here, player 2 must have dealt paper
+
+
+if (hand1 === 'paper') {
+   if (hand2 === 'rock') {
+    return 'Hand one wins!';
+    } else {
+    return 'Hand two wins!';
+   }
+    }
+
+if (hand1 === 'scissors') {
+if (hand2 === 'paper') {
+  return 'Hand one wins!';
+} else {
+return 'Hand two wins!'
```

Comment:
Nicely done!

Please make sure to format this using tab so you're code is easier to read.